DesktopEdge/MainWindow.xaml.cs

```diff
@@ -1514,11 +1518,11 @@ private void SetLocation() {
                 this.Top = desktopWorkingArea.Top + _top;
                 this.Left = desktopWorkingArea.Right - this.Width - _right;
                 Arrow.SetValue(Canvas.TopProperty, (double)0);
-                Arrow.SetValue(Canvas.LeftProperty, (double)185);
+                Arrow.SetValue(Canvas.LeftProperty, (double)195);
```

Comment:
The following was tested to ensure accuracy.  The main window view has an "arrow" square that is aligned to the task bar when the window was "attached".  Originally, it seems like the arrow was aligned where it would most likely point to the corresponding task bar icon.  This will never be achievable in reality because the taskbar size, row stack, and so many other elements could cause the icon to be in a different place.  The result was that the "arrow" just seemed oddly placed.  This fix aligns the arrow back to the app's center (when top/bottom connected to the task bar) and lower portion (when left/right connected to the task bar).  Visually, it doesn't cause an oddity.